evap/locale/de/formats.py

```diff
@@ -1 +1 @@
-from evap.locale.en.formats import *  # pylint: disable=unused-wildcard-import,wildcard-import
+from evap.locale.en.formats import *  # noqa: F403
```

Comment:
Is there any way we can have more readable ignored with ruff? `unused-wildcard-import` says a lot more to me than F403. I guess `undefined-local-with-import-star` should work here?

Reply:
Sadly, nope. There is [astral-sh/ruff#1773](https://www.github.com/astral-sh/ruff/issues/1773) but yeah....

Reply:
Hmm, sad, but I can live with it for now.